### PR TITLE
Added Basic support for Blankom IRD Devices

### PR DESCRIPTION
**Pull request by jozefrebjak**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/bdos.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+os: bdos
+text: ''
```
[the remark this answers is not in the document]
Comment (jozefrebjak) [reply includes/definitions/bdos.yaml:2]: I am planning to add sensors for that device. What is wrong ? OS ? sysobject it's saying bdos, so i used it. That devices are really legacy I think.

File: includes/polling/os/bdos.inc.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?php
+$version = snmp_get($device, '.1.3.6.1.4.1.16744.2.45.1.2.2.0', '-Ovqs', '');
+$serial = snmp_get($device, '.1.3.6.1.4.1.16744.2.45.1.2.13.0', '-Ovqs', '');
+$hardware   = snmp_get($device, '.1.3.6.1.4.1.16744.2.45.1.1.1.0', '-OQv', '');
```
[the remark this answers is not in the document]
Comment (jozefrebjak) [reply includes/polling/os/bdos.inc.php:4]: There was problem to get that info with `snmp_get_multi` from MIB but I will try. This is working for now.

File: includes/definitions/bdos.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheGreatDoc) [reply includes/definitions/bdos.yaml:2]: You mixed comments/reviews.

For this review, you have set `text:` to `''`

About the MIB files, if you plan to add sensor support is ok.

File: includes/polling/os/bdos.inc.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (TheGreatDoc) [reply includes/polling/os/bdos.inc.php:4]: You can use `snmp_get_multi_oid` for numeric oids without MIB but...what issue you do have using the MIB?

**Comment by TheGreatDoc** [comment]
Not sure how, but you had an space at the end of the MIB name

**Comment by TheGreatDoc** [comment]
Oh dude. What a MIB shit. 
They used `_` for declarations and that looks out of the standard.
```
snmpget -v2c -cbdos -M mibs:mibs/blankom -m BLANKOM-DRD700-MIB 127.0.0.1:1161 .
Bad operator (_): At line 43 in mibs/blankom/BLANKOM-DRD700-MIB
ccitt.1 = No Such Instance currently exists at this OID```

My suggestion: try to correct the declarations `_` to `-`. (Only declarations. If you doubt, just run a walk using the MIB and check in what line you get `bad operator` error

**Comment by TheGreatDoc** [comment]
That MIB needs more work.

You only changed declarations, but that is referenced in the MIB. So you need to search for all `dvb_s_s2` and change to `dvb-s-s2` and the same with all of those incorrect declarations

File: includes/definitions/bdos.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jozefrebjak) [reply includes/definitions/bdos.yaml]: @TheGreatDoc added text to Blankom IRD

File: includes/polling/os/bdos.inc.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (jozefrebjak) [reply includes/polling/os/bdos.inc.php]: @TheGreatDoc Changed to `snmp_get_multi_oid` method without MIB

**Comment by jozefrebjak** [comment]
> That MIB needs more work.
> 
> You only changed declarations, but that is referenced in the MIB. So you need to search for all `dvb_s_s2` and change to `dvb-s-s2` and the same with all of those incorrect declarations

I decided to not use MIB for now, I don't have so much time now to play with that. Basic support will be enough for now for us :)

File: includes/polling/os/bdos.inc.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jellyfrog) [reply includes/polling/os/bdos.inc.php]: This file should be converted to the new format. See for example https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Endrun.php

[the remark this answers is not in the document]
Comment (jozefrebjak) [reply includes/polling/os/bdos.inc.php]: @Jellyfrog  I tried, but when I used new format with Number OID's than I can't run successfuly snmpsim. Is there example for Numbered OID's format without MIB ?

[the remark this answers is not in the document]
Comment (Jellyfrog) [reply includes/polling/os/bdos.inc.php]: It should work the same as you're currently doing as far as I know

File: rrd/.gitignore [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # Ignore everything in this directory
 *
 # Except this file
-!.gitignore
```
Comment (murrant) [inline rrd/.gitignore:4]: You edited this file removing the line return. Can you put it back, thanks.

File: includes/polling/os/bdos.inc.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php
```
Comment (murrant) [inline includes/polling/os/bdos.inc.php:1]: This should be in LibreNMS/OS/Bdos.php

**Comment by Jellyfrog** [comment]
Thank you for your contribution but we're closing this due to inactivity.  
Feel free to reopen if you plan on finishing it!

**Pull request closed without merging**